recipe/meta.yaml

```diff
@@ -10,14 +10,18 @@ source:
   sha256: a87c451bf7d4d77aa81c3e6b6b7220fd3ca9fe8e16e303b2e6aeded02cce00ed
 
 build:
-  number: 3
+  number: 4
   skip: True  # [win]
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('fortran') }}
+    - autoconf  # [unix]
+    - gnuconfig  # [unix]
+    - make  # [unix]
   host:
     - pkgconfig
     - readline
```

Comment:
The run section below this can be completely removed. 

Run_exports should be preferred

Reply:
Yes. The packages declared in “host” that have runtime dependencies will declare their runtime dependencies (often the same package itself) with version constraints